### PR TITLE
added sql tables 

### PR DESCRIPTION
**Pull request by MattvanHeusden**
Database table creation

### CODE REVIEW

**Comment by MattvanHeusden** [comment]
@Jason-Joannou

**Comment by Jason-Joannou** [comment]
Hello

File: database/create_tables.py [hunk 1]
```diff
@@ -40,14 +40,15 @@ def create_stokvel_table_sqlite():
...
+                stokvel_id INTEGER PRIMARY KEY, 
+                stokvel_name TEXT UNIQUE,  -- Add UNIQUE constraint here
+                ILP_wallet TEXT,
+                MOMO_wallet TEXT, 
+                total_members INTEGER,
                 min_contributing_amount INTEGER,
                 max_number_of_contributors INTEGER,
-                payout_day INTEGER,
-                created_at TIMESTAMP,
+                Total_contributions FLOAT,
```
Comment (Jason-Joannou) [inline database/create_tables.py:50]: Float needs to be NUMBER

File: database/create_tables.py [hunk 2]
```diff
@@ -40,14 +40,15 @@ def create_stokvel_table_sqlite():
...
+                created_at TIMESTAMP,
```
Comment (Jason-Joannou) [inline database/create_tables.py:51]: Example comment

**Pull request closed without merging**